src/components/sbb-datepicker-toggle/sbb-datepicker-toggle.tsx

```diff
@@ -68,6 +69,17 @@ export class SbbDatepickerToggle implements ComponentInterface {
     }
   }
 
+  /**
+   * Opens the calendar.
+   */
+  @Method()
+  public async open(): Promise<void> {
+    if (!this._triggerElement) {
```

Comment:
How can it be, that the triggerElement is not defined? If you open it before hydrated?

Reply:
Yes, that's the idea, even if I don't really see this happening I thought it wouldn't hurt to cover this scenario

Reply:
It's ok, but unexpected :P